provision-progs.sh

```
@@ -8,14 +8,14 @@ echo "CREATE DATABASE IF NOT EXISTS edraakprograms;" | docker exec -i edx.devsta
 
 
 echo "** Progs: Setting correct python version **"
-docker compose `echo ${DOCKER_COMPOSE_FILES}` exec progs bash -c 'update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1'
+docker compose `echo ${DOCKER_COMPOSE_FILES}` exec progs bash -c 'update-alternatives --install /usr/bin/python python /usr/bin/python3.11 1'
 
 echo "** Progs: Copy cacheed files to code dir **"
 #docker compose `echo ${DOCKER_COMPOSE_FILES}` exec progs bash -c 'cp -Rn /cache/* /app/.'
-docker compose `echo ${DOCKER_COMPOSE_FILES}` exec progs bash -c 'pip3 install -r requirements.txt'
+docker compose `echo ${DOCKER_COMPOSE_FILES}` exec progs bash -c 'pip install -r requirements.txt'
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)